x-pack/plugins/infra/public/pages/logs/stream/page_providers.tsx

```diff
@@ -86,6 +84,13 @@ const LogHighlightsStateProvider: React.FC = ({ children }) => {
 };
 
 export const LogsPageProviders: React.FunctionComponent = ({ children }) => {
+  const { logIndicesExist } = useLogSourceContext();
```

Comment:
Nit: Would it make more sense to use `isLoading` as the condition for short circuiting? (To align with the comment).

Reply:
Hm, but wouldn't that mean that when we reload the source and the indices exist both before and after the loading operation, it would cause significant disruption to the page state? :thinking:

Reply:
Ah I see. Ignore!